scripts/insert_into_appointments.sql

```diff
@@ -1 +1,34 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments (appointment_date,appointment_time,comment,patient_id,doctor_id)
+  SELECT '2016-08-12', '10:00', 'none' p.id, d.id
```

Comment:
This line will cause errors because
- `DATE` needs to precede the string representing the date.
- `TIME` needs to precede the string representing the time.
- there's a missing comma after `'none'`.